integration-cli/docker_cli_links_test.go

```diff
@@ -25,13 +21,9 @@ func (s *DockerSuite) TestLinksInvalidContainerTarget(c *check.C) {
 	testRequires(c, DaemonIsLinux)
 	out, _, err := dockerCmdWithError("run", "--link", "bogus:alias", "busybox", "true")
 
-	if err == nil {
-		c.Fatal("an invalid container target should produce an error")
-	}
-	if !strings.Contains(out, "Could not get container") {
-		c.Fatalf("error output expected 'Could not get container', but got %q instead; err: %v", out, err)
-	}
-
+	// an invalid container target should produce an error
+	c.Assert(err, checker.NotNil)
```

Comment:
Please include `out` in ` check.Commentf` here, so that if a failure occurs we can see what happened, otherwise we'd generally just get `exit status 1` or something from the err.